migration-engine/migration-engine-tests/tests/single_migration_tests/postgres/enum_basic.prisma

```diff
@@ -1,6 +1,11 @@
 // tags=postgres
 // exclude=cockroachdb
 
+datasource testds {
+    provider = "postgresql"
+    url = env("TEST_DATABASE_URL")
+}
+
```

Comment:
Apparently the Postgres CI would fail without this `datasource` block. I think this test was added a couple of days ago in https://github.com/prisma/prisma-engines/pull/3338.

Reply:
Indeed, this is failing on main. I think it happened because Julius' introspection model rendering PR wasn't rebased, and with an older `main` these tests worked without a datasource.

Could you make this change into a separate PR so we can get this fixed asap? I'm happy to do it too if you don't have time.

Reply:
https://github.com/prisma/prisma-engines/pull/3348